<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalc</class>
 <widget class="QStackedWidget" name="CreditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
background: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="Form">
   <widget class="QPushButton" name="toCalc">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
border: 1px solid rgb(36, 48, 94);
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
    </property>
    <property name="text">
     <string>Calc</string>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>20</y>
      <width>205</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>24</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
border-radius: 10px;
background: rgb(236, 240, 249);
color: rgb(36, 48, 94);
font: 24pt &quot;.AppleSystemUIFont&quot;;
}</string>
    </property>
    <property name="text">
     <string>Loan Calculator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Amount">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
border-radius: 10px;
background: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
    </property>
    <property name="text">
     <string>Loan Amount</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Interest">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
border-radius: 10px;
background: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
    </property>
    <property name="text">
     <string>Interest Amount</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="Term">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
border-radius: 10px;
background: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
    </property>
    <property name="text">
     <string>Loan Term</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="amountSlider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove {
background-color: rgb(36, 48, 94);
height: 7px;
border-radius: 10px;
}

QSlider::handle {
width: 15px;
margin: -3px 0;
background-color: #FFF
}</string>
    </property>
    <property name="minimum">
     <number>100000</number>
    </property>
    <property name="maximum">
     <number>1000000000</number>
    </property>
    <property name="singleStep">
     <number>100000</number>
    </property>
    <property name="pageStep">
     <number>100000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Sign">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resource.qrc">:/images/resource/sign.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="toDepos">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>290</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
border: 1px solid rgb(36, 48, 94);
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
    </property>
    <property name="text">
     <string>Depos</string>
    </property>
   </widget>
   <widget class="QSlider" name="interestSlider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove {
background-color: rgb(36, 48, 94);
height: 7px;
border-radius: 10px;
}

QSlider::handle {
width: 15px;
margin: -3px 0;
background-color: #FFF
}</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="termSlider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove {
background-color: rgb(36, 48, 94);
height: 7px;
border-radius: 10px;
}

QSlider::handle {
width: 15px;
margin: -3px 0;
background-color: #FFF
}</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>36</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="loanEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>115</y>
      <width>170</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
border-radius: 10px;
background-color: #FFF;
color: rgb(36, 48, 94);
padding: 10px;
}</string>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="interestEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>255</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
border-radius: 10px;
background-color: #FFF;
color: rgb(36, 48, 94);
padding: 10px;
}</string>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="termEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>185</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
border-radius: 10px;
background-color: #FFF;
color: rgb(36, 48, 94);
padding: 10px;
}</string>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="calculate">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
border: 1px solid rgb(36, 48, 94);
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkM">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
spacing: 5px;
color: rgb(36, 48, 94);
}

QCheckBox::indicator {
width: 15px;
height: 15px;
background-color: rgb(236, 240, 249);
}

QCheckBox::indicator:unchecked {
    image: url(:/images/resource/Vector 2.svg);
}

QCheckBox::indicator:checked {
border: none;
image: url(:/images/resource/Group 8.svg);
}</string>
    </property>
    <property name="text">
     <string>Months</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkY">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>205</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
spacing: 5px;
color: rgb(36, 48, 94);
}

QCheckBox::indicator {
width: 15px;
height: 15px;
background-color: rgb(236, 240, 249);
}

QCheckBox::indicator:unchecked {
    image: url(:/images/resource/Vector 2.svg);
}

QCheckBox::indicator:checked {
border: none;
image: url(:/images/resource/Group 8.svg);
}</string>
    </property>
    <property name="text">
     <string>Years</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkAnn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
spacing: 5px;
color: rgb(36, 48, 94);
}

QCheckBox::indicator {
width: 15px;
height: 15px;
background-color: rgb(236, 240, 249);
}

QCheckBox::indicator:unchecked {
    image: url(:/images/resource/Vector 2.svg);
}

QCheckBox::indicator:checked {
border: none;
image: url(:/images/resource/Group 8.svg);
}</string>
    </property>
    <property name="text">
     <string>Annuity</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkDiff">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>135</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
spacing: 5px;
color: rgb(36, 48, 94);
}

QCheckBox::indicator {
width: 15px;
height: 15px;
background-color: rgb(236, 240, 249);
}

QCheckBox::indicator:unchecked {
    image: url(:/images/resource/Vector 2.svg);
}

QCheckBox::indicator:checked {
border: none;
image: url(:/images/resource/Group 8.svg);
}</string>
    </property>
    <property name="text">
     <string>Differentiated</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Result">
   <widget class="QLabel" name="Sign_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resource.qrc">:/images/resource/sign.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
border: 1px solid rgb(36, 48, 94);
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="op">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
border-radius: 10px;
background: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
    </property>
    <property name="text">
     <string>Overpayment:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
border-radius: 10px;
background: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
    </property>
    <property name="text">
     <string>Total payment:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="overPay">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
border-radius: 10px;
background: #FFF;
color: rgb(36, 48, 94);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="monPay">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>340</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
border-radius: 10px;
background: #FFF;
color: rgb(36, 48, 94);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
border-radius: 10px;
background: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
    </property>
    <property name="text">
     <string>Monthly payment:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="totPay">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
border-radius: 10px;
background: #FFF;
color: rgb(36, 48, 94);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
